slave-nodejs6-centos7/test/run

```diff
@@ -10,5 +10,6 @@ docker run --rm -t --entrypoint=/bin/bash ${IMAGE_NAME} -ic 'source /usr/local/b
 docker run --rm -t --entrypoint=/bin/bash ${IMAGE_NAME} -ic 'source /usr/local/bin/configure-slave && git --version'
 docker run --rm -t --entrypoint=/bin/bash ${IMAGE_NAME} -ic 'source /usr/local/bin/configure-slave && npm --version'
 docker run --rm -t --entrypoint=/bin/bash ${IMAGE_NAME} -ic 'source /usr/local/bin/configure-slave && node --version'
+docker run --rm -t --entrypoint=/bin/bash ${IMAGE_NAME} -ic 'source /usr/local/bin/configure-slave && openshift-template-tool version'
```

Comment:
This will return non-zero in the case that it's just the html redirect info, and not the expected binary.